static/docker-compose.yml

```diff
@@ -315,6 +328,33 @@ services:
     entrypoint: npm run
     command: "start:contextual-scoring-service"
 
+  ckeditor-backend:
+    deploy:
+      replicas: 3
+      update_config:
+        parallelism: 1
+    image: docker.cke-cs.com/cs:4.17.1
+    depends_on:
+      redis:
+        condition: service_started
+      postgres:
+        condition: service_healthy
+    restart: always
+    init: true
+    environment:
+      DATABASE_DRIVER: postgres
+      DATABASE_HOST: postgres
+      DATABASE_PORT: 5432
+      DATABASE_USER: ${PG_CKEDITOR_ADMIN_USER}
+      DATABASE_PASSWORD: ${PG_CKEDITOR_ADMIN_PASSWORD}
+      DATABASE_DATABASE: ${PG_CKEDITOR_DB}
+      DATABASE_SCHEMA: public
+      REDIS_HOST: redis
+      REDIS_CONNECTION_STRING: redis://redis:6379
```

Comment:
I think this may also be a different connection string unless a different redis is being stood up